Client/Frontend/Sync/SyncSettingsTableViewController.swift

```diff
@@ -114,8 +177,9 @@ class SyncSettingsTableViewController: UITableViewController {
     
     override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
         let deviceCount = frc?.fetchedObjects?.count ?? 0
+        let buttonsCount = 1
         
-        return section == Sections.buttons.rawValue ? ButtonsSection.count : deviceCount
+        return section == Sections.buttons.rawValue ? buttonsCount : deviceCount
```

Comment:
Just to clarify, I just meant a normal comment, but I see what's going on here. No need for the constant here, I think previously was clear enough. But I understand better what is going on. 

Not going to block PR on this. Just wanted to leave a comment.